cfme/infrastructure/config_management.py

```diff
@@ -181,18 +413,22 @@ def delete(self, cancel=False, wait_deleted=True, force=False):
         """
         if not force and not self.exists:
             return
-        navigate_to(self, 'All')
-        sel.check(Quadicon(self.quad_name, None).checkbox())
+        view = navigate_to(self, 'All')
+        if not view.toolbar.view_selector.is_displayed:
+            print('Selector not displayed, trying to refresh browser')
```

Comment:
**REQUIRED:** We should rather use the logger here

Reply:
Whoops, didn't mean to leave that in. Will remove it.